src/Microsoft.AspNet.Mvc.ApiExplorer/DefaultApiDescriptionProvider.cs

```diff
@@ -302,6 +313,56 @@ private string GetRelativePath(RouteTemplate parsedTemplate)
             return string.Join("/", segments);
         }
 
+        private IReadOnlyList<ApiRequestFormat> GetRequestFormats(
+            ControllerActionDescriptor action,
+            IApiRequestMetadataProvider[] requestMetadataAttributes,
+            Type type)
+        {
+            var results = new List<ApiRequestFormat>();
```

Comment:
Move the declaration closer to it's first usage

You can also return null instead of the empty list in case there are no results, in other areas of the code we do that and it only adds an if clause to check for null in https://github.com/aspnet/Mvc/pull/3892/files#diff-0e55710732ebb46d94a4706b0b9c24b7R143